VocabularyTrainer/Localizable/Localizable.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Localizable.swift
+//  VocabularyTrainer
+//
+//  Created by Mariana Brasil on 16/12/22.
+//  Copyright © 2022 mic. All rights reserved.
+//
+
+import UIKit
+
+enum Localizable: String {
```

Comment:
Thanks for thinking about the localized strings as well!

However, I guess this could get a really huge enum at some point if it's global. Or do you plan of having these on a screen basis?

What do you think of changing to MVVM in general starting with this screen? 

Then we could make these Localizable enums an extension to a new `NewLanguageViewModel`. Going a little bit away from the massive view controllers now, will be easier to maintain in the future I think.

A slightly different approach (that I'm currently using in other projects) is making all of the strings static and use an `enum` just for namespacing. Adding the `context` parameter can be valuable for translations/documentation later on. We should consider using it as well. You could use the same approach for any static data like `Images` and  `URLs`.

```swift
extension NewLanguageViewModel {
    enum Strings {
        static let title = NSLocalizedString(
            "title_text",
            value: "The title",
            comment: "Text for the title"
        )
        static let buttonTitle = NSLocalizedString(
            "button_title",
            value: "I am a button",
            comment: "Title of the button"
        )
    }
}
```

in the view controller we could add `typealias`es to make the calling site nicer:

```swift
class NewLanguageViewController: UIViewController {

    typealias Strings = NewLanguageViewModel.Strings

    let viewModel: NewLanguageViewModel
    let button = UIButton()

    init(viewModel: NewLanguageViewModel) {
        self.viewModel = viewModel
    }

    func setUI() {
        button.setTitle(Strings.buttonTitle, for: .normal)
    }
}
```

Let me know what you think :) We can also do the MVVM refactoring in another MR but I think we should do it.